src/main/java/lotto/domain/RankMatches.java

```diff
@@ -1,31 +1,41 @@
 package lotto.domain;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public enum RankMatches {
-    FIRST(6, 2000000000),
-    SECOND(5, 1500000),
-    THIRD(4, 50000),
-    FOURTH(3, 5000),
+    FIRST(6, 2_000_000_000),
+    SECOND(5, 30_000_000),
+    THIRD(5, 1_500_000),
+    FOURTH(4, 50_000),
+    FIFTH(3, 5_000),
     NONE(0, 0);
 
     private static int MINIMUM_OF_PRIZE = 3;
     private int count;
     private int prize;
+    private static final Map<Integer, RankMatches> rankMap = new HashMap<>();
+
+    static{
+        rankMap.put(6, FIRST);
+        rankMap.put(5, THIRD);
+        rankMap.put(4, FOURTH);
+        rankMap.put(3, FIFTH);
+    }
```

Comment:
```suggestion
    private static final Map<Integer, RankMatches> rankMap = 
        Arrays.stream(this.values())
            .filter(rank -> rank != NONE)
            .collect(Collectors.toMap(
                rank -> rank.count,
                Function.identity()
            );
```

이런 방식으로도 작성이 가능합니다.
등수가 추가될 일은 없지만 만약 추가되었을 때 static 영역을 잊는다면 오류가 발생할 수 있겠네요.

이런 방식으로도 작성할 수 있습니다.
getCount 메소드는 없으므로

Reply:
앗 그렇군요! static이라 this는 못써서 약간 변형하여 수정했습니다! 감사합니다 :)